.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,21 @@
-<!--
-****Please describe the pull request motivation and changes here
-along with non-obvious things.
--->
+## Overview
+
+<!-- Please describe your changes here 
+  and list any open questions you might have. -->
+
+---
+<!-- This is for Exonum Team members only. -->
+<!-- markdownlint-disable MD034 -->
+See: https://jira.bf.local/browse/ECR-XYZW
+<!-- markdownlint-enable MD034 -->
+
+### Definition of Done
+
+- [ ] There are no TODOs left in the merged code
+- [ ] Change is covered by automated tests
+- [ ] The [coding guidelines] are followed
+- [ ] Public API has proper documentation
+- [ ] Changelog is updated if needed (in case of notable or breaking changes)
+- [ ] The continuous integration build passes
```

Comment:
One more thing which may come in handy in this check-list:

```
- [ ] Benchmark results are attached (if applicable, in case the change is performance-related)
```

Reply:
Slightly changed this line (because of 80-symbols line length restriction)